docker-compose.yml

```diff
@@ -1,26 +1,59 @@
 version: '3.6'
 services:
   hasura:
-    image: hasura/graphql-engine:v2.0.4
+    container_name: hasura
+    build:
+      context: .
+      dockerfile: Dockerfile.hasura    
     ports:
-    - "8080:8080"
+      - "8080:8080"
     restart: always
-    environment:
-      ## this env var can be used to add the above postgres database to Hasura as a data source. this can be removed/updated based on your needs
+    environment:      
       HASURA_GRAPHQL_DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/${POSTGRES_DB}
```

Comment:
Can we use another port? Since `postgres` is running by default in `5432`, if we have a local instance running we can have a conflict with the docker one.